force-app/main/default/classes/UTIL_HtmlOutput_CTRL.cls

```diff
@@ -53,7 +53,7 @@ public with sharing class UTIL_HtmlOutput_CTRL {
 
```

Comment:
```suggestion
        '<a' => '|anchor|'
```
@andrewyu-salesforce / @daniel-fuller this would be the change I'm referring to in relation to the broken test.

Reply:
The other option is to directly replace the powerofus URL with ```<a href="https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000kHitSAE``` as in this case we can't just use a dummy URL and would need to reference a URL that is an allowed URL